Doc/library/math.rst

```diff
@@ -41,12 +41,15 @@ Number-theoretic and representation functions
    Return the number of ways to choose *k* items from *n* items without repetition
    and without order.
 
-   Also called the binomial coefficient. It is mathematically equal to the expression
-   ``n! / (k! (n - k)!)``. It is equivalent to the coefficient of the *k*-th term in the
-   polynomial expansion of the expression ``(1 + x) ** n``.
+   Evaluates to ``n! / (k! * (n - k)!)`` when ``k <= n`` and evaluates
+   to zero when ``k > n``.
 
-   Raises :exc:`TypeError` if the arguments not integers.
-   Raises :exc:`ValueError` if the arguments are negative or if *k* > *n*.
+   Also called the binomial coefficient because it is equivalent
+   to the coefficient of k-th term in polynomial expansion of the
+   expression ``(1 + x) ** n``.
+
+   Raises :exc:`TypeError` if either of the arguments not integers.
```

Comment:
Typo (predates this PR): missing "are" before "not integers". Same again below for math.perm. I'd personally  reword as "... unless both arguments are integers".

Reply:
Good catch.